app/assets/css/variables.styl

```diff
@@ -0,0 +1,41 @@
+// Colors
+$primary = #00b092
+$primary-shade = darken($primary, 10)
+$light-gray = #f2f2f2
+$tertiary = #f5f5f5
+$dark-gray  = #706d6e
+$border-color = #d1d1d1
+$font-primary = #333
+$font-secondary = #666
+$drug-color = #097BBD // Analogous of primary. See https://color.adobe.com/
+
+// UI
+$block-margins = 0 0 20px 0
+$padding-size = 20px
+$container-max-width = 750px
+
+// Material shadows
+$shadow-key-umbra-opacity = 0.2
+$shadow-key-penumbra-opacity = 0.14
+$shadow-ambient-shadow-opacity = 0.12
+
+$shadow-focus = 0 0 8px rgba(0,0,0,.18),0 8px 16px rgba(0,0,0,.36);
+$shadow-2dp = 0 2px 2px 0 rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 3px 1px -2px rgba(0, 0, 0, $shadow-key-umbra-opacity), 0 1px 5px 0 rgba(0, 0, 0, $shadow-ambient-shadow-opacity)
+$shadow-3dp = 0 3px 4px 0 rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 3px 3px -2px rgba(0, 0, 0, $shadow-key-umbra-opacity), 0 1px 8px 0 rgba(0, 0, 0, $shadow-ambient-shadow-opacity)
+$shadow-4dp = 0 4px 5px 0 rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 1px 10px 0 rgba(0, 0, 0, $shadow-ambient-shadow-opacity), 0 2px 4px -1px rgba(0, 0, 0, $shadow-key-umbra-opacity)
+$shadow-6dp = 0 6px 10px 0 rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 1px 18px 0 rgba(0, 0, 0, $shadow-ambient-shadow-opacity), 0 3px 5px -1px rgba(0, 0, 0, $shadow-key-umbra-opacity)
+$shadow-8dp = 0 8px 10px 1px rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 3px 14px 2px rgba(0, 0, 0, $shadow-ambient-shadow-opacity), 0 5px 5px -3px rgba(0, 0, 0, $shadow-key-umbra-opacity)
+$shadow-16dp = 0 16px 24px 2px rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0  6px 30px 5px rgba(0, 0, 0, $shadow-ambient-shadow-opacity), 0  8px 10px -5px rgba(0, 0, 0, $shadow-key-umbra-opacity)
+$shadow-24dp = 0 9px 46px  8px rgba(0, 0, 0, $shadow-key-penumbra-opacity), 0 11px 15px -7px rgba(0, 0, 0, $shadow-ambient-shadow-opacity), 0 24px 38px  3px rgba(0, 0, 0, $shadow-key-umbra-opacity)
+
+$shadow-transition = all 0.3s cubic-bezier(.25,.8,.25,1)
+
+// Mixins
+noselect()
```

Comment:
added this mixin from aliem... nice for turning off select highlights from buttons and such -- not using it yet